lib/core.js

```diff
@@ -85,9 +92,9 @@ function createProcessor(options = {}) {
       .use(rehypePlugins)
       .use(rehypeRecma)
       .use(recmaDocument, {...otherOptions, contain})
-      // @ts-ignore recma transformer uses an estree node rather than a unist node
+      // @ts-ignore recma transformer uses an esast node rather than a unist node
       .use(recmaJsxRewrite, {providerImportSource, contain})
-      // @ts-ignore recma transformer uses an estree node rather than a unist node
+      // @ts-ignore recma transformer uses an esast node rather than a unist node
```

Comment:
thinking of making a “spec” for this: estree but with `.position` and `.data` (for which I’d still need to write adapters to convert between the two, for `.position`)

Reply:
gotcha, would it make sense to stylize the name a bit `EsAst`, `esAst`, `ESast`, or `esAST`?
`esast` makes me do a double take when I'm skimming/speed reading as it's similar to `east`.

Reply:
I was thinking either esast or east, similar to mdast/hast.

I somewhat strongly dislike capitalizing them 😅 (although links/strong where possible are :+1: from me)